### PR TITLE
DellEMC S6000 : Platform2.0 API implementation [PSU, Thermal]

### PR DESCRIPTION
**Pull request by ArunSaravananBalachandran**
**- What I did**

Implement PSU and Thermal platform2.0 API for DellEMC S6000 platform

**- How I did it**

Added new files psu.py and thermal.py for PSU and thermal platform2.0 API implementation

**- How to verify it**

Wrote python script to load Chassis class and then call PSU and Thermal APIs accordingly.
UT logs : [UT-logs_PSU.txt](https://github.com/Azure/sonic-buildimage/files/3509230/UT-logs_PSU.txt) , [UT-logs_thermal.txt](https://github.com/Azure/sonic-buildimage/files/3509232/UT-logs_thermal.txt)

**- Description for the changelog**

DellEMC S6000 : Platform2.0 API implementation [PSU, Thermal]

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: platform/broadcom/sonic-platform-modules-dell/s6000/sonic_platform/psu.py [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+        Sets the state of the PSU status LED
+        Args:
+            color: A string representing the color with which to set the
+                   PSU status LED
+        Returns:
+            bool: True if status LED state is set successfully, False if
+                  not
+        """
+        # Hardware not supported
+        return False
```
Comment (jleveque) [inline platform/broadcom/sonic-platform-modules-dell/s6000/sonic_platform/psu.py]: The S6000 has status LEDs on each PSU and also one power status LED on the front panel. How are these controlled?

Comment (ArunSaravananBalachandran) [reply platform/broadcom/sonic-platform-modules-dell/s6000/sonic_platform/psu.py]: In S6000, the firmware running in the PSU controls the individual PSU's status LED.
The power status LED on the front panel is controlled via CPLD.

Comment (jleveque) [reply platform/broadcom/sonic-platform-modules-dell/s6000/sonic_platform/psu.py]: So the CPLD will manage the power status LED on the front panel without software assistance?

Comment (ArunSaravananBalachandran) [reply platform/broadcom/sonic-platform-modules-dell/s6000/sonic_platform/psu.py]: The front panel power status LED needs to be controlled by software via CPLD.

Comment (jleveque) [reply platform/broadcom/sonic-platform-modules-dell/s6000/sonic_platform/psu.py]: OK. Do you intend to control the power status LED somewhere other than here?

Comment (ArunSaravananBalachandran) [reply platform/broadcom/sonic-platform-modules-dell/s6000/sonic_platform/psu.py]: Currently, the front panel power status LED is not controlled here. If required we can take this as separate work-item.

**Comment by sujinmkang** [comment]
@ArunSaravananBalachandran can you please resolve the conflicts?

**Comment by ArunSaravananBalachandran** [comment]
> @ArunSaravananBalachandran can you please resolve the conflicts?

Merge conflicts have been resolved.

**Pull request merged**